# This expresses a marker/fiducial detection
Header header                  # timestamp in the header is the acquisition time and frame

MarkersAvg[] markers               # detected markers